usaspending_api/recipient/delta_models/recipient_profile.py

```diff
@@ -1,26 +1,436 @@
+                OR rp.recipient_name IS DISTINCT FROM temp_p.recipient_name
+                OR rp.recipient_affiliations IS DISTINCT FROM temp_p.recipient_affiliations
+                OR rp.award_types IS DISTINCT FROM temp_p.award_types
+                OR rp.last_12_months IS DISTINCT FROM temp_p.last_12_months
+                OR rp.last_12_contracts IS DISTINCT FROM temp_p.last_12_contracts
+                OR rp.last_12_loans IS DISTINCT FROM temp_p.last_12_loans
+                OR rp.last_12_grants IS DISTINCT FROM temp_p.last_12_grants
+                OR rp.last_12_direct_payments IS DISTINCT FROM temp_p.last_12_direct_payments
+                OR rp.last_12_other IS DISTINCT FROM temp_p.last_12_other
+                OR rp.last_12_months_count IS DISTINCT FROM temp_p.last_12_months_count
+                OR rp.uei IS DISTINCT FROM temp_p.uei
+                OR rp.parent_uei IS DISTINCT FROM temp_p.parent_uei)
+        WHEN MATCHED THEN UPDATE SET
+            recipient_unique_id = temp_p.recipient_unique_id,
+            uei = temp_p.uei,
+            recipient_name = temp_p.recipient_name,
+            recipient_affiliations = temp_p.recipient_affiliations,
+            award_types = temp_p.award_types,
+            last_12_months = temp_p.last_12_months,
+            last_12_contracts = temp_p.last_12_contracts,
+            last_12_loans = temp_p.last_12_loans,
+            last_12_grants = temp_p.last_12_grants,
+            last_12_direct_payments = temp_p.last_12_direct_payments,
+            last_12_other = temp_p.last_12_other,
+            last_12_months_count = temp_p.last_12_months_count,
+            parent_uei = temp_p.parent_uei
+        WHEN NOT MATCHED THEN
+        INSERT (
+            id,
+            recipient_level,
+            recipient_hash,
+            recipient_unique_id,
+            uei,
+            parent_uei,
+            recipient_name,
+            recipient_affiliations,
+            award_types,
+            last_12_months,
+            last_12_contracts,
+            last_12_loans,
+            last_12_grants,
+            last_12_direct_payments,
+            last_12_other,
+            last_12_months_count
+        ) VALUES (
+            temp_p.id + 1,
+            temp_p.recipient_level,
+            temp_p.recipient_hash,
+            temp_p.recipient_unique_id,
+            temp_p.uei,
+            temp_p.parent_uei,
+            temp_p.recipient_name,
+            temp_p.recipient_affiliations,
+            temp_p.award_types,
+            temp_p.last_12_months,
+            temp_p.last_12_contracts,
+            temp_p.last_12_loans,
+            temp_p.last_12_grants,
+            temp_p.last_12_direct_payments,
+            temp_p.last_12_other,
+            temp_p.last_12_months_count
+        );
+    """,
+    """DROP TABLE temporary_restock_recipient_profile;""",
+    """DROP VIEW temporary_recipients_from_transactions_view;""",
```

Comment:
Don't think you need to drop the view since it will only exist for the duration of the session.

Reply:
Also, might need to add the DB you are dropping the table from depending on the response to question regarding where the table is created.

Reply:
removed the table, need to drop view to prevent errors with the tests.